draft-ietf-tls-tls13.md

```diff
@@ -5475,16 +5475,16 @@ TLSPlaintext.legacy_record_version MUST be ignored by all implementations.
 The value of TLSCiphertext.legacy_record_version MAY be ignored,
 or MAY be validated to match the fixed constant value.
 Version negotiation is performed using only the handshake versions
-(ClientHello.legacy_version,
-ClientHello "supported_versions" extension, and ServerHello.version).
+(ClientHello.legacy_version, as well as
+ClientHello and ServerHello "supported_versions" extension).
```

Comment:
"as well as the ClientHello and ServerHello "supported_versions" extensions".
It's not clear whether the HelloRetryRequest specialization of ServerHello requires a separate callout here.

Reply:
@kaduk isn't `extension` (singular) the correct form? We're talking about the same extension, not multiple ones. Your observation applies to my other edit as well, I made it explicit in the previous case, so I'll add it here too.

Reply:
It's kind of subtle.  To start off with, we're talking about version negotiation, which means we care about the contents of the extension blocks, not the type off extension.  And there are distinct instances of the same extension type in ClientHello and ServerHello, potentially holding different contents (used for negotiation).  So the presence of different instances justifies the use of the plural, to me.

Reply:
@kaduk yeah, I see where you're coming from. Actually since we're on it, shouldn't I add the `ServerHello.legacy_version` to the text too? I missed that one.

Reply:
@kaduk I added `ServerHello.legacy_version`, as well as accepted your grammar suggestion in 0ef3784